Fixed typos and grammar issues (#160)
aws/eks-anywhere: release/runbook.md

# EKS Anywhere Release Runbook

> EKS Anywhere releases are of two variants: the versioned bundle release and the EKS-A CLI release.

Reviewer: ```suggestion
EKS Anywhere releases are of two variants: the versioned bundle release and the EKS Anywhere CLI release.
```

### Versioned Bundles Release

> The versioned bundles are comprised of version-tagged bundles of EKS-A/EKS-D components. This includes the list of container images/manifests from EKS-A, as well as EKS-D dependent artifacts such as OVAs and kinD images. These image and S3 URIs will be embedded into a bundles manifest that the CLI will use to fetch the images and manifests that it needs to pull when creating a cluster. The bundles release manifest URI will be referenced in the EKS-A CLI release manifest, so that customers will know what version bundle each release version of the CLI will be supporting.

Reviewer: ```suggestion
The versioned bundles are comprised of version-tagged bundles of EKS Anywhere and EKS Distro components. This includes the list of container images/manifests from EKS Anywhere, as well as EKS Distro-dependent artifacts such as OVAs and kinD images. These image and S3 URIs will be embedded into a bundles manifest that the CLI will use to fetch the images and manifests that it needs to pull when creating a cluster. The bundles release manifest URI will be referenced in the EKS Anywhere CLI release manifest, so that customers will know what version bundle each release version of the CLI will be supporting.
```

> ### EKS-A CLI release

Reviewer: ```suggestion
### EKS Anywhere CLI release
```

> The EKS-A CLI release is the build and release of the tagged version of the EKS-A CLI, along with a reference to the corresponding bundle release manifest. The staging EKS-A release will be kicked off after the staging bundle release, so that this release manifest can reference a bundle manifest that exists in the artifacts S3. After the staging release has been uploaded to S3, we can do integration tests in staging by obtaining the CLI from the release manifest and running it against the staging versioned bundles. Once the tests pass, all the manifests and artifacts can be moved from staging to prod.

Reviewer: ```suggestion
The EKS Anywhere CLI release is the build and release of the tagged version of the EKS Anywhere CLI, along with a reference to the corresponding bundle release manifest.
The staging EKS Anywhere release will be kicked off after the staging bundle release, so that this release manifest can reference a bundle manifest that exists in the artifacts S3.
After the staging release has been uploaded to S3, we can do integration tests in staging by obtaining the CLI from the release manifest and running it against the staging versioned bundles.
Once the tests pass, all the manifests and artifacts can be moved from staging to prod.
```
We're trying to start every sentence on its own line. I haven't been particularly strict on this, but I think it's something we should aspire to.
